tasks/build/javascript.js

```diff
@@ -46,19 +46,16 @@ const
  * @param config
  * @return {*}
  */
-function build(src, type, config) {
+function build(src, type, compress, config) {
   return gulp.src(src)
     .pipe(plumber())
     .pipe(flatten())
     .pipe(replace(comments.license.in, comments.license.out))
-    .pipe(gulpif(config.hasPermissions, chmod(config.parsedPermissions)))
-    .pipe(gulp.dest(config.paths.output.uncompressed))
-    .pipe(print(log.created))
```

Comment:
Shouldn't this be kept? It might be needed to do the uglify and compress.. The destination path is now only changed after the possible compression, when this is removed

Otherwise there will always be the same path for compressed/uncompressed . (Line 58)
This is probably always the case anyway, but to follow the possible different paths we should keep it backward compatible....Or am i missunderstanding the logic here ? @ColinFrick Any advise?

Reply:
This should be fine.